doc/2/api/controllers/document/search/index.md

```diff
@@ -50,6 +50,15 @@ Body:
 }
 ```
 
+You can also access this route with the `GET` verb:
+
+```http
+URL: http://kuzzle:7512/<index>/<collection>/_search[?query=<string>][?from=<int>][&size=<int>][&scroll=<time to live>][&verb=<string>]
```

Comment:
Ditto for the `verb` query string argument

Reply:
same as above